extensions/adobe/experience/decisioning/proposition-event-type.schema.json

```diff
@@ -0,0 +1,48 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/proposition-event-type",
```

Comment:
Are you defining a data type here ? Looks like yes as I see that you have referenced this in the "propositionEventType" property in the [experienceevent-proposition-interaction.schema.json] field group.

If this is a data type, please remove meta:intendedToExtend as that is only to define field group and its association to a class.

Reply:
Yes. I have removed the metadata field.